scripts/gh/overview-dag.sh

```
@@ -0,0 +1,123 @@
+tmpdir=$(mktemp -d)
+trap 'rm -rf "$tmpdir"' EXIT
+
+echo "Fetching issues from $REPO ..." >&2
+if ! gh issue list --repo "$REPO" --state open -L 300 --json number,title,labels,url > "$tmpdir/issues.json"; then
+  echo "ERROR: failed to list issues for $REPO" >&2
+  exit 1
+fi
+
+# Get epic checklist bodies (we use these to infer epic→child edges)
+epics=$(jq -r '.[] | select(.title|test("^Epic:")) | .number' "$tmpdir/issues.json")
+>"$tmpdir/edges.txt"
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)